src/perftest_parameters.h

```diff
@@ -439,6 +439,7 @@ struct perftest_parameters {
 	int				disable_fcs;
 	int				flows;
 	int				flows_burst;
+        int                             wait_keystroke;
```

Comment:
I don't need this change except for system testing. So unless you deem it interesting for general use, feel free to drop it.